common/src/main/java/com/wynntils/screens/maps/MainMapScreen.java

```diff
@@ -230,6 +233,31 @@ public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialT
                         .pointerType
                         .get());
 
+        LootrunInstance currentLootrun = Models.Lootrun.getCurrentLootrun();
+
+        if (currentLootrun != null) {
+            MapRenderer.renderLootrunLine(
+                    currentLootrun,
+                    3f,
+                    poseStack,
+                    centerX,
+                    centerZ,
+                    mapCenterX,
+                    mapCenterZ,
+                    currentZoom,
+                    CommonColors.BLACK.asInt());
+            MapRenderer.renderLootrunLine(
+                    currentLootrun,
+                    2f,
+                    poseStack,
+                    centerX,
+                    centerZ,
+                    mapCenterX,
+                    mapCenterZ,
+                    currentZoom,
+                    CommonColors.LIGHT_BLUE.asInt());
```

Comment:
I would at least like to merge these calls, so we don't do most of the work twice. After that, I am most likely fine with performance.

If you don't have time, I can patch it up and merge it. Otherwise, thanks.